apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Using specific templates in PDPs and PLPs'
+sidebar_label: 'Multiple page templates'
+---
+
+<header>
+
+# Multiple page templates
+
+</header>
+
+Multiple Page Template means that PDPs and PLPs, previously defined as singleton in the CMS, can now assume different page templates based on some rules. This approach is almost similar to the PDP and PLP, but they will be explained separately since each page type has particularities.
+
+## PDP Template
+
+When rendering the PDPs, FastStore will use the product information to find the appropriate document from the CMS (template) to render, starting from the most specific template (slug) of the product to the most generic (`department` or the generic PDP).
+
+The order should be as follows:
+
+1.  A PDP document that matches the product slug;
+2.  A PDP document that matches the product `subcategory`;
+3.  A PDP document that matches the product `category`;
+4.  A PDP document that matches the product `department` (collection);
+5.  If no matches are found, use the generic PDP template.
+
+![PDP document list](https://vtexhelp.vtexassets.com/assets/docs/src/pdplist___a4862453821a4f753ac1d171cc3708b6.png)
```

Comment:
nitpick (non-blocking): bilingual..🙈 Can we retake the prints? I think it's better to keep everything in English.

<img width="828" alt="image" src="https://github.com/vtex/faststore/assets/3356699/d1daa483-4ff4-4dcc-9994-51a687adcb26">

Reply:
The prints were changed [here](https://github.com/vtex/faststore/pull/2236/commits/459dc70dc998371c07a705565fc5145a8fb833e3). ty